diff --git a/dist/acp.js b/dist/acp.js
index 05048c2dc9859d7d240ca3f2647512ba486776eb..604be3854471e6878736993090923974b39d0e4e 100644
--- a/dist/acp.js
+++ b/dist/acp.js
@@ -431,7 +431,7 @@ export class ClientSideConnection {
      * Select a model for a given session.
      */
     async setSessionModel(params) {
-        return ((await this.#connection.sendRequest(schema.AGENT_METHODS.session_set_mode, params)) ?? {});
+        return ((await this.#connection.sendRequest(schema.AGENT_METHODS.session_set_model, params)) ?? {});
     }
     /**
      * Authenticates the client using the specified authentication method.
